src/Security/Authentication/JwtBearer/src/JwtBearerHandler.cs

```diff
@@ -143,6 +143,9 @@ protected override async Task<AuthenticateResult> HandleAuthenticateAsync()
                             SecurityToken = validatedToken
                         };
 
+                        tokenValidatedContext.Properties.ExpiresUtc = validatedToken?.ValidTo;
+                        tokenValidatedContext.Properties.IssuedUtc = validatedToken?.ValidFrom;
```

Comment:
validatedToken is not null after ValidateToken.

Reply:
Must be a test only thing then, is it easy to construct a fake one?

Reply:
? The test doesn't look like it's using a fake validator. It looks like validatedToken is never used except for the event, maybe it really could be null. That's worth spending 5 minutes to debug, check which validator type is used, and if it can return null on success.

Reply:
https://github.com/dotnet/aspnetcore/blob/eba50232b08c526cfe19f7a9097ce2a8469a2b13/src/Security/Authentication/test/JwtBearerTests.cs#L934
I think I can just set it to a `TestSecurityToken`

Reply:
This test isn't using the fake BlobTokenValidator...

Reply:
Updated

Reply:
:shipit: